### DISCUSSION TITLE
Unable to receive my Student Github benefits

### DISCUSSION THREAD

**Discussion by yakimk** [community/community#69201] [GitHub Education]
### Hi everyone,

Hi, I applied for github student pack twice already, but cannot get accepted. My student id has expiring date on the back side and my name appears only at the front. First time when I took pictures of both sides and glued them into one, my application got rejected, because it "appears to be edited". But there is no way I could send one photo of the front and the back of my ID without editing two pictures together.
Any suggestions?

**Comment by stolenfallen1** [comment]
Hi yakimk, 

Have you tried Alternative Verification? Like using study load? or try resubmit your application using your ID but include a note explaining your situation for context.

**Reply by yakimk** [reply]
Yep. Unfortunately adding an explanation didn't help, it got rejected for the same reasons and since I don't pay student loans, I don't really have a clear way to prove my current student status except for a student ID.
